scripts/doc_crosslinks/generate_cross_link_data.py

```diff
@@ -14,22 +15,17 @@
 
 def generateDocsMap(apiDefinitionsPath, apiDefinitionsRelativeFilePath):
 
-    filesInDir = [f for f in listdir(apiDefinitionsPath) if isdir(join(apiDefinitionsPath, f))]
-    for file in filesInDir :
-        serviceJsonFileName = join(apiDefinitionsPath, join(file, apiDefinitionsRelativeFilePath))
-
-        if(exists(serviceJsonFileName)) :
-            with codecs.open(serviceJsonFileName, 'rb', 'utf-8') as api_definition:
-                api_content = json.loads(api_definition.read())
-                if "uid" in api_content["metadata"].keys():
-                    sdks[api_content["metadata"]["uid"]] = file
-                clientClass[api_content["metadata"]["uid"]] = getClientClassNameFromMetadata(api_content["metadata"])
-
-#                 # Below  code can be used for debugging  failing clients
-#                 str  = "https://sdk.amazonaws.com/java/api/latest/software/amazon/awssdk/services/"+ file +"/"+getClientClassNameFromMetadata(api_content["metadata"])+".html"+"#validateTemplate--"
-#                 ret = requests.head(str)
-#                 if( ret.status_code != 200 ):
-#                     print(str)
+    root = pathlib.Path(r'./services')
+    for filepath in root.rglob('service-2.json'):
+        tmpArr = str(Path(filepath).parent).split("/")
+        getName = tmpArr[len(tmpArr)-1]
+        if (getName == "codegen-resources"):
+            getName = str(filepath).split("services/")[1].split("/src/main/resources")[0]
```

Comment:
Can we rename this variable name so it is easy it to identify what it signifies.
In this case it could be moduleName